### PR TITLE
Emily design

### PR DESCRIPTION
**Pull request by brunoolmedo**
added register button to logistics section, added sketch licenses award to design section, minor text edits to "what is polyhack" paragraph.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -185,7 +180,7 @@ <h3 id="mission">Polyhack is one of the few hackathons that allows people to wor
 		<div class="row tealbg" id="Designers">
 			<div class="col-sm-7 col-sm-push-5 right">
 				<h1 class="right-title">Designers</h1>
-				<h3 id="mission">Polyhack is the only hackathon that runs a series of sponsored design competitions, also known as the <a href="http://go.tufts.io/design-wars">Design Wars</a>, as part of the event. You can work with a hackathon team and/or work on the design prompts all night long. After reviewing all the design submissions at the end of the day, the top 3 winners will get portfolio reviews with either Frog, Palantir, or Jacob Waites (who is an Interaction Designer at IDEO CoLAB).</h3>
+				<h3 id="mission">Polyhack is the only hackathon that runs a series of sponsored design competitions, also known as the <a href="go.tufts.io/mentor">Design Wars</a>, as part of the event. You can work with a hackathon team and/or work on the design prompts all night long. After reviewing all the design submissions at the end of the day, the top 3 winners will get Sketch licenses and portfolio reviews with either Frog, Palantir, or Jacob Waites (who is an Interaction Designer at IDEO CoLAB).</h3>
```
Comment (cwRichardKim) [inline index.html]: you need http:// for the href to work. else it googles it or adds it to the end of the website name

Comment (cwRichardKim) [reply index.html]: also why is this being changed from design wars to mentors when it's in the design wars section?

**Review by tekknolagi — APPROVED** [review]
lgtm

File: index.html (hunk 1 above)
Comment (tekknolagi) [reply index.html]: @cwRichardKim it was an old ref that I just fixed

**Pull request merged**